infra/gcp/ensure-staging-storage.sh

```diff
@@ -205,6 +208,7 @@ function ensure_staging_gcr_repo() {
     fi
     local project="${1}"
     local writers="${2}"
+    local gcr_bucket="artifacts.${1}.appspot.com"
```

Comment:
Given my other comments, I would tack gs:// onto this for consistency
```suggestion
    local gcr_bucket="gs://artifacts.${1}.appspot.com"
```